README.md

```diff
@@ -58,11 +58,14 @@ curl -L https://coder.com/install.sh | sh -s -- --help
 Once installed, you can start a production deployment with a single command:
 
 ```sh
-# Automatically sets up an external access URL on *.try.coder.app
+# Optionally sets up an external access URL on *.try.coder.app
 coder server --tunnel
 
-# Requires a PostgreSQL instance and external access URL
-coder server --postgres-url <url> --access-url <url>
+# Using a PostgreSQL instance and external access URL
+coder server --postgres-url <url> [--access-url <url>]
+
+# OR using embedded PostgreSQL (not recommended for production)
```

Comment:
Why isn't this recommended for production?

Reply:
Uhhhh... you know, I didn't really think very hard about that.  I guess it's not about production so much as the *scale* of production that might be an issue.  I can take it out no prob.